packages/rn-tester/RNTesterPods.xcodeproj/project.pbxproj

```diff
@@ -589,7 +589,7 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/sh;
-			shellScript = "export NODE_BINARY=node\nexport PROJECT_ROOT=$SRCROOT/..\nexport SOURCEMAP_FILE=../sourcemap.ios.map\n# export FORCE_BUNDLING=true\n\"$SRCROOT/../../scripts/react-native-xcode.sh\" $SRCROOT/../../packages/rn-tester/js/RNTesterApp.ios.js\n";
+			shellScript = "export NODE_BINARY=node\nexport SOURCEMAP_FILE=../sourcemap.ios.map\n# export FORCE_BUNDLING=true\n\"$SRCROOT/../../scripts/react-native-xcode.sh\" $SRCROOT/../../packages/rn-tester/js/RNTesterApp.ios.js\n";
```

Comment:
@alloy, I see that you have added this in the previous PR (that you did). 

Ultimately, you're overwriting PROJECT_ROOT in order to correctly read HERMES_PATH from root `node_modules`. I decided to slightly update the script to make this one unnecessary. 

We have this attitude inside CLI (and generally, when it comes to scripts) to avoid setting paths that are related to project roots etc., to not complicate the setup for monorepo users.

Reply:
Is this going to be a script that people have to manually modify in their xcode project UI? If so, should this be a separate script that is executed from the step to make future project upgrades easier?

Reply:
Having less such variables is indeed a good goal, but please also look at https://github.com/facebook/react-native/pull/29477 and see if that can be consolidated.

Reply:
Yes. Sounds good! I am keeping that default assingement inside the `react-native-xcode.sh`. Just removed setting it in RNTester itself, where it's not necessary.

Reply:
Changes inside #29477 are not yet in this PR, but there should be no conflict between these two (eventually, we need to find what broke the fix in that PR)

Reply:
I am resolving this based on your +1